articles/article-047-game-of-life.md

```diff
@@ -0,0 +1,27 @@
+<!-- BEGIN TITLE -->
+# Game of life
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+John Horton Conway invented a simple game with a small amount of rules. 
+Each cell on a square grid is either dead or alive. For each iteration the cell looks at it eight neighbors.
```

Comment:
For each iteration the cell looks at its eight neighbors. 
- reason its as a possessive (we don’t use the apostrophe, it's means it is)